project 3/CODE/index.html

```diff
@@ -0,0 +1,140 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css?family=Quicksand|Ubuntu:700" rel="stylesheet">
+    <title>Timeline</title>
+</head>
+
+<body>
+
+    <div id="hour6am">
```

Comment:
I think you could simplify your CSS by changing up your markup a bit:
```html
    <div class="timeline-item">
        <a href="#hour6am-content"><img src="images/6am.jpg" alt=''></a>
        <div id="hour6am-content" class="timeline-content">
            <a href="#" class="button-close">&times;</a>
            <h2>6:00 AM</h2>
            <p>Here is a constant debate of whether to get up or sleep for another 15 minutes and then get up. This is really tough for me.</p>
        </div>
    </div>
```
With this markup, instead of having to target all the individual elements you want to hide/be revealed, you can just target the `div` containing them:
```css
.timeline-content {
    display: none;
}

.timeline-content:target {
    display: block;
}
```